src/index.ts

```diff
@@ -60,6 +73,7 @@ const plugin: JupyterFrontEndPlugin<void> = {
             'jupyterlab_run_and_reload settings loaded:',
             settings.composite
           );
+          settings.changed.connect(loadSetting);
         })
         .catch(reason => {
           console.error(
```

Comment:
is it possible to inject a logger so that this error event is persistent somewhere?

Reply:
Made #13 to add a logger later.